hbase_storage/hbase1.1.0/src/main/java/com/splicemachine/access/client/MemStoreFlushAwareScanner.java

```diff
@@ -140,6 +151,8 @@ public boolean internalNext(List<Cell> outResult,ScannerContext scannerContext)
                 }
             } else {
                 flushAlreadyReturned = true;
+                if (LOG.isTraceEnabled())
+                    SpliceLogUtils.trace(LOG, "Next: returning begin flush ");
```

Comment:
Why this change is not applied to 1.0.0.x and 1.0.0?

Reply:
They weren't too important and the commits will have to be reverted or cleaned up after debugging the wrong result in jenkins, but I cleaned up this piece and added the missing logs. Thanks for the review!